model_zoo/census_model_sqlflow/wide_and_deep/wide_deep_functional_tensor_interface_fc.py

```diff
@@ -0,0 +1,315 @@
+import tensorflow as tf
+from tensorflow import feature_column as fc
+
+from elasticdl.python.elasticdl.callbacks import LearningRateScheduler
+from elasticdl_preprocessing.feature_column import feature_column as edl_fc
+from model_zoo.census_model_sqlflow.wide_and_deep.feature_configs import (
+    FEATURE_TRANSFORM_INFO_EXECUTE_ARRAY,
+    INPUT_SCHEMAS,
+    TRANSFORM_OUTPUTS,
+    age_bucketize,
+    capital_gain_bucketize,
+    capital_loss_bucketize,
+    education_hash,
+    group1_embedding_deep,
+    group1_embedding_wide,
+    group2_embedding_deep,
+    group2_embedding_wide,
+    group3_embedding_deep,
+    hours_per_week_bucketize,
+    marital_status_lookup,
+    native_country_hash,
+    occupation_hash,
+    race_lookup,
+    relationship_lookup,
+    sex_lookup,
+    workclass_lookup,
+)
+from model_zoo.census_model_sqlflow.wide_and_deep.transform_ops import (
+    TransformOpType,
+)
+
+
+# The model definition from model zoo. It's functional style.
+# Input Params:
+#   input_layers: The input layers dict of feature inputs
+#   wide_embedding: A tensor. Embedding for the wide part.
+#   deep_embedding: A tensor. Embedding for the deep part.
+def wide_and_deep_classifier(input_layers, wide_embedding, deep_embedding):
+    # Wide Part
+    wide = wide_embedding  # shape = (None, 3)
+
+    # Deep Part
+    dnn_input = deep_embedding
+    for i in [16, 8, 4]:
+        dnn_input = tf.keras.layers.Dense(i)(dnn_input)
+
+    # Output Part
+    concat_input = tf.concat([wide, dnn_input], 1)
+
+    logits = tf.reduce_sum(concat_input, 1, keepdims=True)
+    probs = tf.reshape(tf.sigmoid(logits), shape=(-1,))
+
+    return tf.keras.Model(
+        inputs=input_layers,
+        outputs={"logits": logits, "probs": probs},
+        name="wide_deep",
+    )
+
+
+# Build the input layers from the schema of the input features
+def get_input_layers(input_schemas):
+    input_layers = {}
+
+    for schema_info in input_schemas:
+        input_layers[schema_info.name] = tf.keras.layers.Input(
+            name=schema_info.name, shape=(1,), dtype=schema_info.dtype
+        )
+
+    return input_layers
+
+
+# Build the transform logic from the metadata in feature_configs.py.
+def transform(inputs):
+    feature_column_dict = {}
+
+    for feature_transform_info in FEATURE_TRANSFORM_INFO_EXECUTE_ARRAY:
```

Comment:
Is `FEATURE_TRANSFORM_INFO_EXECUTE_ARRAY` some data only used for testing? In real cases, does `FEATURE_TRANSFORM_INFO_EXECUTE_ARRAY` should be generated by the `COLUMN` clause? Also the `TRANSFORM_OUTPUTS`.

Reply:
FEATURE_TRANSFORM_INFO_EXECUTE_ARRAY and TRANSFORM_OUTPUTS won't exist in the generated code.
I think it's the intermediate object during code generation. It should be exist in GO.